contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -294,6 +298,28 @@ change_key(pam_handle_t *pamh, const char *ds_name,
 		zfs_close(ds);
 		return (-1);
 	}
+	keyformat = zfs_prop_get_int(ds, ZFS_PROP_KEYFORMAT);
```

Comment:
```suggestion
	uint64_t keyformat = zfs_prop_get_int(ds, ZFS_PROP_KEYFORMAT);
```

We do allow declaring the variable when first used.

Reply:
will fix in next revision